src/main/java/org/team1540/robot2023/Constants.java

```diff
@@ -166,6 +166,8 @@ public static final class ArmConstants {
         public static final double MAX_LEGAL_DISTANCE = 48 + PIVOT_DISTANCE;
         // The maximum distance the arm will point to
         public static final double MAX_POINT_DISTANCE = MAX_LEGAL_DISTANCE + 100; // TODO: 1/30/2023 this should be the maximum limelight detection distance
+
+        public static final double PIVOT_TICKS_TO_DEGREES = 0;// TODO: 2/1/2023 figure this out so that position things work
```

Comment:
Might want to rename to "PIVOT_DEGREES_PER_TICK" to make the units of this constant more clear